wolfcrypt/src/rsa.c

```diff
@@ -1624,6 +1624,9 @@ static int RsaUnPad(const byte *pkcsBlock, unsigned int pkcsBlockLen,
         word16 j;
         word16 pastSep = 0;
 
+        if (pkcsBlockLen > 0xFFFF)
```

Comment:
If possible please move this up to the check at the beginning of the function. It looks like possibly okay to return here, but would be better to return where there is already an exit to the function in places that are critical to run in constant time.